src/Pos/Balances.hs

```diff
@@ -0,0 +1,59 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module Pos.Balances
```

Comment:
I think this should be somewhere is `Pos.Txp`, like, `Pos.Txp.MonadBalances`. I don't like this module is not related to any subpart, like `Txp` or `Wallet`